index.html

```diff
@@ -7,7 +7,7 @@
     <style>
       * {font-family: 'Open Sans', 'Segoe UI'; margin: 0; padding: 0; list-style: none; font-weight: normal;}
       html, body {height: 100%;}
-      body {display: flex; justify-content: center; align-items: center; flex-direction: column; height: 100%; background: url("https://i.imgur.com/gnoq6cwl.png") center center; -webkit-background-size: cover; background-size: cover;}
+      body {color : white ; background:black ;display: flex; justify-content: center; align-items: center; flex-direction: column; height: 100%; background: url("https://i.imgur.com/gnoq6cwl.png") center center; -webkit-background-size: cover; background-size: cover;}
```

Comment:
Do not use `white` but consider using `#fff`. You've used `black` but it's actually half black.